exercises/word-search/src/test/java/WordSearcherTest.java

```diff
@@ -67,6 +280,53 @@ public void testLocatesWordsWrittenRightToLeft() {
         assertEquals(expectedLocations, actualLocations);
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void testShouldLocateASingleWordRightToLeft() {
```

Comment:
I think the previous test should be removed? It appears to now be both before and after this test 🙂

Reply:
Oops! Fixed up now 🙂